app/src/main/java/com/danielkashin/yandexweatherapp/view/main_drawer/ITitleContainer.java

```diff
@@ -0,0 +1,8 @@
+package com.danielkashin.yandexweatherapp.view.main_drawer;
+
+
+public interface ITitleContainer {
```

Comment:
Some java style guides recommend to place I before interfaces

I my previous project toolbar also contained some icons, and I passed boolean paramethers from fragment to activity so that activity knows what icons to show. Is that an anti-pattern?